posix-configs/bebop/mainapp.config

```diff
@@ -4,9 +4,10 @@ param load
 param set SYS_AUTOSTART 4001
 param set MAV_BROADCAST 1
 df_ms5607_wrapper start
+df_mpu6050_wrapper start
```

Comment:
No, it is not. I prepared a fix within another future pull-request. But I will add it here.